app/server/appsmith-plugins/redisPlugin/src/main/java/com/external/plugins/RedisPlugin.java

```diff
@@ -125,6 +138,43 @@ public RedisPlugin(PluginWrapper wrapper) {
                     .subscribeOn(scheduler);
         }
 
+        private boolean isQueryFormatValid(Map cmdAndArgs) {
+            if (cmdAndArgs.containsKey(CMD_KEY)) {
+                return true;
+            }
+
+            return false;
```

Comment:
Nit: This function can be simplified into a single line statement.
```suggestion
            return cmdAndArgs.containsKey(CMD_KEY);
```